extensions/in-memory/dataaddress-resolver-memory/build.gradle.kts

```diff
@@ -22,8 +22,8 @@ dependencies {
 }
 publishing {
     publications {
-        create<MavenPublication>("in-memory.assetindex") {
-            artifactId = "in-memory.assetindex"
+        create<MavenPublication>("in-memory.dataadressresolver") {
+            artifactId = "in-memory.dataadressresolver"
```

Comment:
Raised upstream in https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/203